main_menu.py

```diff
@@ -111,7 +111,6 @@ def __init__(self):
         self.my_pet_screen = PetSelection(pygame, screen)
         self.my_pet_screen.initialize_pets()
         self.my_rock = PetRock(input_pygame=pygame, screen=screen)
-        self.my_rock.set_current_animation(config.RockActions.dying.value)
```

Comment:
I removed this because in the `house_screen.py` the `my_rock` animation is also set.